src/app/categories/state/entry-categories.service.ts

```diff
@@ -60,5 +59,8 @@ export class EntryCategoriesService {
           this.entryCategoriesStore.setError(true);
         }
       );
+    } else {
+      this.entryCategoriesStore.set(null);
```

Comment:
The `remove()` above will remove everything from the categories store, this line may not be necessary.

Reply:
Confirmed that this suggestion works.